Fix handling of quota on volumes (#975)
containers/storage: docs/containers-storage.conf.5.md

Now all new content created in these directories will automatically be created with the correct label.

## QUOTAS

> Container storage impliments `XFS project quota controls for overlay storage

Reviewer: ```suggestion
Container storage implements `XFS project quota controls for overlay storage
```

containers and volumes. The directory used to store the containers must be an
`XFS` file system and be mounted with the `pquota` option.

Example /etc/fstab entry:
```
/dev/podman/podman-var /var xfs defaults,x-systemd.device-timeout=0,pquota 1 2
```

> Container storage generates project ids for each container and builtin volume, but these project ids need to be unigue for the XFS file system.

Reviewer 2: ```suggestion
Container storage generates project ids for each container and builtin volume, but these project ids need to be unique for the XFS file system.
```

> xfs_quota tool can be used to assign a project id to the storage driver directory, e.g.:

Reviewer 2: ```suggestion
The xfs_quota tool can be used to assign a project id to the storage driver directory, e.g.:
```

```

> echo 100000:/var/lib/containers/storage/storage >> /etc/projects

Reviewer 3: ```suggestion
echo 100000:/var/lib/containers/storage/overlay >> /etc/projects
```

echo 200000:/var/lib/containers/storage/volumes >> /etc/projects
echo storage:100000 >> /etc/projid
echo volumes:200000 >> /etc/projid
xfs_quota -x -c 'project -s storage volumes' /<xfs mount point>
```	     

> In that case, the storage directory project id will be used as a "start offset"

Reviewer 2: ```suggestion
In the example above, the storage directory project id will be used as a "start offset"
```

and all containers will be assigned larger project ids (e.g. >= 100000).

> the volumes directory project id will be used as a "start offset"

Reviewer 2: ```suggestion
Then the volumes directory project id will be used as a "start offset"
```
